WalletWasabi/WabiSabi/Backend/WabiSabiConfig.cs

```diff
@@ -30,13 +30,34 @@ public WabiSabiConfig(string filePath) : base(filePath)
 	[JsonProperty(PropertyName = "ConfirmationTarget", DefaultValueHandling = DefaultValueHandling.Populate)]
 	public uint ConfirmationTarget { get; set; } = 108;
 
-	[DefaultValueTimeSpan("0d 3h 0m 0s")]
-	[JsonProperty(PropertyName = "ReleaseUtxoFromPrisonAfter", DefaultValueHandling = DefaultValueHandling.Populate)]
-	public TimeSpan ReleaseUtxoFromPrisonAfter { get; set; } = TimeSpan.FromHours(3);
+	[DefaultValueMoneyBtc("1.0")]
+	[JsonProperty(PropertyName = "DoSSeverity", DefaultValueHandling = DefaultValueHandling.Populate)]
+	[JsonConverter(typeof(MoneyBtcJsonConverter))]
+	public Money DoSSeverity { get; set; } = Money.Coins(0.1m);
 
 	[DefaultValueTimeSpan("31d 0h 0m 0s")]
-	[JsonProperty(PropertyName = "ReleaseUtxoFromPrisonAfterLongBan", DefaultValueHandling = DefaultValueHandling.Populate)]
-	public TimeSpan ReleaseUtxoFromPrisonAfterLongBan { get; set; } = TimeSpan.FromDays(31);
+	[JsonProperty(PropertyName = "DoSMinTimeForFailedToVerify", DefaultValueHandling = DefaultValueHandling.Populate)]
+	public TimeSpan DoSMinTimeForFailedToVerify { get; set; } = TimeSpan.FromDays(31);
+
+	[DefaultValueTimeSpan("1d 0h 0m 0s")]
+	[JsonProperty(PropertyName = "DoSMinTimeForCheating", DefaultValueHandling = DefaultValueHandling.Populate)]
+	public TimeSpan DoSMinTimeForCheating { get; set; } = TimeSpan.FromDays(1);
+
+	[DefaultValue(1.0)]
+	[JsonProperty(PropertyName = "DoSPenaltyFactorForDisruptingConfirmation", DefaultValueHandling = DefaultValueHandling.Populate)]
+	public double DoSPenaltyFactorForDisruptingConfirmation { get; set; } = 1.0;
+
+	[DefaultValue(1.5)]
+	[JsonProperty(PropertyName = "DoSPenaltyFactorForDisruptingSigning", DefaultValueHandling = DefaultValueHandling.Populate)]
+	public double DoSPenaltyFactorForDisruptingSigning { get; set; } = 1.5;
+
+	[DefaultValue(1.5)]
+	[JsonProperty(PropertyName = "DoSPenaltyFactorForDisruptingByDoubleSpending", DefaultValueHandling = DefaultValueHandling.Populate)]
+	public double DoSPenaltyFactorForDisruptingByDoubleSpending { get; set; } = 3.0;
```

Comment:
Same here. `[DefaultValue(1.5)]`, but `3.0` in the code.

Reply:
fixed